server/team-validator-async.js

```diff
@@ -88,7 +89,7 @@ if (!PM.isParentProcess) {
 		});
 	}
 
-	global.Dex = require('../sim/dex').includeData();
+	global.Dex = (/** @type {typeof import('../sim/dex')} */(require(/** @type {any} */ ('../.sim-dist/dex')))).includeData();
```

Comment:
```suggestion
	global.Dex = require(/** @type {any} */ ('../.sim-dist/dex')).includeData();
```

The outer cast is unnecessary; the type is defined in `globals.ts`.